packages/components/src/Input/InputSearch/_docs/InputSearch.stickersheet.stories.tsx

```diff
@@ -0,0 +1,177 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { InputSearch } from "../index"
+
+export default {
+  title: "KAIO-staging/Inputs/InputSearch",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+    a11y: {
+      config: {
+        rules: [
+          {
+            // Built with no label on purpose, to be used within `TextField` where label is present
+            id: "label",
+            enabled: false,
+          },
+        ],
+      },
+    },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: ({ isReversed }) => (
+    <StickerSheet isReversed={isReversed}>
+      <StickerSheet.Header
+        headings={["Default", "Disabled", "Hover", "Focus"]}
```

Comment:
I feel like we could also have a "filled" or "active" column  with a `value` passed in so we can have a snapshot of the search input with a clear button